io_export_cryblend/export.py

```diff
@@ -195,9 +195,23 @@ def __export_library_images(self, parent_element):
             self.__convert_images_to_dds(images)
 
     def __export_library_image(self, image):
+        relative_path = bpy.path.abspath(self.__config.filepath)
+        last_slash = relative_path.rfind("\\")
+        if last_slash != -1:
+            relative_path = relative_path[:last_slash]
+

```

Comment:
Modified already the three path calculation. Seems to work for me, please review it again.